File: sources/services/Theta.Services/Theta.ItineraryService/Models/Activity.cs
using System;

namespace Theta.ItineraryService.Models
{
    public class Activity
    {
        public int Id { get; set; }

        public DateTime? From { get; set; }

        public DateTime? To { get; set; }

        public Location Location { get; set; }

        public string[] Tags { get; set; }

        public string Description { get; set; }
    }
}

File: project/theta_app/src/app/itineraries/models/activity.ts
import { BaseModel } from 'src/app/models/base-model';
import { Place } from './place';
import { Image } from '../../models/image';

export interface Activity extends BaseModel {
  id: number;
  activity: string;
  day: number;
  startTime: string;
  endTime: string;
  tags: string;
  price: number;
  description: string;
  sequence: number;
  place: Place;
  images: Image[];
}
